app/views/partials/_nav.html.erb

```diff
@@ -18,8 +18,9 @@
       <% end %>
     <% else %>
       <div class="align-middle table-cell p3">
-        <%= link_to 'Sign In', login_path, class: "px2"%>
-        <%= link_to 'Register', register_path %>
+        <%= link_to 'Sign In', login_path, class: "px2" %>
+        <%= link_to 'Register', register_path, class: "px2" %>
+        <%= link_to 'About', about_path, class: "px2" %>
```

Comment:
`class: "px2"` on only one link made the spacing asymmetrical, so i gave them all spacing.